app/src/main/res/values/strings.xml

```diff
@@ -176,26 +176,26 @@
     <string name="vivo_keypress_workaround">vivo Origin OS keypress workaround</string>
     <string name="invalid_im">Invalid input method configuration file:\n%1$s</string>
     <string name="invalid_table_dict">Invalid dictionary:\n%1$s</string>
-    <string name="exception_table_im">Table input method configuration file (.conf or .conf.in) not found</string>
-    <string name="exception_table">Table dictionary (.dict or .txt) not found</string>
-    <string name="table_im">Table Input Methods</string>
-    <string name="manage_table_im">Manage Table Input Methods</string>
+    <string name="exception_table_im">Configuration file of table-based input method (.conf or .conf.in) not found</string>
+    <string name="exception_table">Mapping table dictionary (.dict or .txt) not found</string>
+    <string name="table_im">Table-based Input Methods</string>
+    <string name="manage_table_im">Manage Table-based Input Methods</string>
     <string name="exception_table_im_filename">The extension of \"%1$s\" does not indicate a table input method package (.zip)</string>
-    <string name="table_already_exists">Table input method configuration file \"%1$s\" already exists</string>
-    <string name="table_file_does_not_exist_title">Table unavailable</string>
-    <string name="table_file_does_not_exist_message">Table dictionary \"%1$s\" does not exist. Please check the dict file or re-import.</string>
-    <string name="import_table">Import Table</string>
+    <string name="table_already_exists">Configuration file of table-based input methods \"%1$s\" already exists</string>
+    <string name="table_file_does_not_exist_title">Character mapping table unavailable</string>
+    <string name="table_file_does_not_exist_message">Mapping table dictionary \"%1$s\" does not exist. Please check the dict file or re-import.</string>
```

Comment:
The name should be consistent. But I would rather keep "Table dictionary" rather than  "Character mapping table" or "Mapping table dictionary".